packages/manager-components/src/components/templates/onboarding/onboarding.component.tsx

```diff
@@ -55,38 +58,62 @@ export const OnboardingLayout: React.FC<OnboardingLayoutProps> = ({
             color={ODS_THEME_COLOR_INTENT.primary}
             level={ODS_TEXT_LEVEL.heading}
             size={ODS_TEXT_SIZE._700}
-            className="block text-center mb-3"
+            className="block text-center mb-4"
           >
             {title}
           </OsdsText>
-          <OsdsText
-            level={ODS_TEXT_LEVEL.body}
-            className="block text-center mb-2"
-          >
-            {description}
-          </OsdsText>
-          <div className="flex sm:py-8 xs:w-full xs:flex-col sm:items-center sm:w-fit sm:flex-row">
-            <div className="w-full xs:py-3">
+          {typeof description === 'string' ? (
+            <OsdsText
+              level={ODS_TEXT_LEVEL.body}
+              size={ODS_TEXT_SIZE._800}
+              className="block text-center mb-2"
```

Comment:
Why `mb-2` and not `mb-4` ? Margin-bottom value will defer regarding if `description` is an array or a string, is it weird right ?

Reply:
Adjusted for mb-4